setup.py

```diff
@@ -76,12 +76,17 @@ def read_to_rst(fname):
 #This detects if we are building on a Mac
 def ismac():
     return sys.platform[:6] == 'darwin'
+def iswin():
+    return sys.platform[:5] == 'win32'
 
 if ismac():
     compile_flags = [ '-O3', ]
     link_args = []
+elif iswin():
+    compile_flags = [ '/openmp', '/O3']
```

Comment:
Slightly surprised that MS VC++ doesn't also require a compile flag for debugging. It's been about 17 years since I specified a debug option, but I'm happy to see from https://docs.microsoft.com/en-us/cpp/build/reference/z7-zi-zi-debug-information-format?view=vs-2019 that the compile options haven't changed. You probably want `/ZI` or `/Zi`.